shell/common/api/electron_api_asar.cc

```diff
@@ -52,8 +52,8 @@ class Archive : public node::ObjectWrap {
       return;
     }
 
-    auto archive = std::make_unique<asar::Archive>(path);
-    if (!archive->Init()) {
+    auto archive = asar::GetOrCreateAsarArchive(path);
```

Comment:
prefer to avoid `auto` unless the type name is given in full elsewhere on the same line:

```suggestion
    asar::Archive archive = asar::GetOrCreateAsarArchive(path);
```